.dockerignore

```diff
@@ -0,0 +1 @@
+**/appsettings.Development.json
```

Comment:
does it also include the stuff in .gitignore?

Reply:
Don't think it does. Should copy the same things to this file.

Reply:
I don't think it needs the whole gitignore, but it *should* exclude `appsettings.Local.json`.